src/render.rs

```diff
@@ -1529,8 +1528,7 @@ impl<'a> TerminalRenderer {
                 }
             }
             (_, Key::Down) => {
-                if self.app.selected_process.is_some() {
```

Comment:
Makes sense. Had ideas about what the arrows would down when focused on a process, haven't got around to implementing them.